addon/globalPlugins/charinfo/__init__.py

```diff
@@ -29,11 +29,12 @@
 import globalVars
 import config
 from utils import security
-
-# Temporarily import private function as advised by Sean Budd from NVAccess.
-# A public API can be requested by add-on developers 2-3 month after 2022.3.3 release if no issue is found.
-from winAPI.sessionTracking import _isLockScreenModeActive
-
+try
+	# For NVDA >= 2023.2
+	from winAPI.sessionTracking import isLockScreenModeActive as isLockScreen
```

Comment:
Could you just used the new symbol defined in NVDA's core without renaming? It's quite clear and easier to find in the code.

```suggestion
	from winAPI.sessionTracking import isLockScreenModeActive
```

Of course, it should be renamed to `isLockScreenModeActive` in the except clause and in the rest of the code.